src/main/webapp/app/exercises/shared/plagiarism/exercise-update-plagiarism/exercise-update-plagiarism.component.html

```diff
@@ -12,8 +12,8 @@
                     id="continuous_plagiarism_control_enabled"
                     class="plagiarism-option-checkbox form-check-input"
                     name="exercise.plagiarismDetectionConfig.continuousPlagiarismControlEnabled"
-                    #exercise.plagiarismDetectionConfig.continuousPlagiarismControlEnabled="ngModel"
-                    [(ngModel)]="exercise.plagiarismDetectionConfig!.continuousPlagiarismControlEnabled"
+                    [checked]="exercise.plagiarismDetectionConfig.continuousPlagiarismControlEnabled"
+                    (click)="toggleCPCEnabled()"
```

Comment:
Why this is needed?

Reply:
Using a ngModel is not possible here since I want to toggle the after due date checks as well, see #7611